operator/pkg/validate/common.go

```diff
@@ -143,6 +145,45 @@ func validatePortNumber(path util.Path, val interface{}) util.Errors {
 	return validateIntRange(path, val, 0, 65535)
 }
 
+// validateGlobalLoggingLevel validates global loggin level and also allow empty, examples: "default:debug", ""
+func validateGlobalLoggingLevel(path util.Path, val interface{}) (errs util.Errors) {
+	scope.Debugf("validateGlobalLoggingLevel at %v: %v", path, val)
+
+	if !util.IsString(val) {
+		err := fmt.Errorf("validateGlobalLoggingLevel %s got %T, want string", path, val)
+		printError(err)
+		return util.NewErrs(err)
+	}
+
+	if val.(string) == "" {
+		return nil
+	}
+
+	lvc := strings.Split(val.(string), ":")
+	if len(lvc) != 2 || lvc[0] != "default" {
+		err := fmt.Errorf("validateGlobalLoggingLevel got logging level scope '%s', want 'default' scope across all components", lvc[0])
```

Comment:
why? this seems pretty arbitrary... if they user wants it why stop them? If you want to "solve" the issue reported, the logging library should just allow unknown scope names in my opinion - because this doesn't actually resolve that issue, it just makes it fail slightly sooner